### PR TITLE
Estimating the portfolio error

### PR DESCRIPTION
**Pull request by micheles**
This is tentative and may be refined later on. Merging so that we can use it and have an idea of how well it works.

### CODE REVIEW

File: openquake/calculators/views.py [hunk 1]
```diff
@@ -362,8 +368,12 @@ def view_portfolio_loss(token, dstore):
     oq = dstore['oqparam']
     G = getattr(oq, 'number_of_ground_motion_fields', 1)
     R = dstore['full_lt'].get_num_rlzs()
-    means = dstore['losses_by_event']['loss'].sum(axis=0) / R / G
-    return rst_table([means], oq.loss_names)
+    loss = dstore['losses_by_event']['loss']  # shape (E, L)
+    means = loss.sum(axis=0) / R / G
+    sums = [loss[idxs].sum(axis=0) for idxs in _indices(len(loss), 10)]
+    errors = numpy.std(sums, axis=0) / numpy.mean(sums, axis=0) * means
```
Comment (raoanirudh) [inline openquake/calculators/views.py:374]: Couple of comments: 
1. The mean loss for the portfolio that we report in the avg_losses files is a weighted average of the realizations, while here we are reporting a simple average of the realizations. This could cause confusion – if a user adds up the avg_losses for all assets from the csv export, it will not match the mean portfolio loss reported here.
2. What's the error that's being reported on this line?

Comment (raoanirudh) [reply openquake/calculators/views.py:374]: Ok, I get it now. This code splits the losses by event into blocks of 10 events, and computes the standard deviation of the total losses from each of those blocks to provide a kind of measure of the variance in the losses to understand if the number of events is sufficient to achieve reasonable convergence on the average loss values.

But we can even use a simpler measure, i.e., the standard error: https://en.wikipedia.org/wiki/Standard_error

Comment (raoanirudh) [reply openquake/calculators/views.py:374]: Ideally, since we wish to report just a single mean value and single standard error for the calculation, we should calculate the average loss and standard error for each realization separately, then report the weighted mean for both using the weights for each realization.
